apis/v1alpha1/db_cluster.go

```diff
@@ -815,6 +858,16 @@ type DBClusterStatus struct {
 	// restore.
 	// +kubebuilder:validation:Optional
 	LatestRestorableTime *metav1.Time `json:"latestRestorableTime,omitempty"`
+	// Contains the secret managed by RDS in Amazon Web Services Secrets Manager
+	// for the master user password.
+	//
+	// For more information, see Password management with Amazon Web Services Secrets
+	// Manager (https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/rds-secrets-manager.html)
+	// in the Amazon RDS User Guide and Password management with Amazon Web Services
+	// Secrets Manager (https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/rds-secrets-manager.html)
+	// in the Amazon Aurora User Guide.
+	// +kubebuilder:validation:Optional
+	MasterUserSecret *MasterUserSecret `json:"masterUserSecret,omitempty"`
```

Comment:
I wonder if this will be confusing since the term `Secret` could refer to a K8s `Secret` too?

Reply:
Agreed. OTOH this is the name used in the upstream API, so someone referencing the RDS API docs would look for this. Need to think on this.

Reply:
I think the Go docs (which will make its way into the API reference on the website) do a pretty good of explaining this